lib/player.py

```diff
@@ -752,7 +752,8 @@ def _playVideo(self, offset=0, seeking=0, force_update=False, playerObject=None)
 
         meta = self.playerObject.metadata
 
-        url = meta.streamUrls[0]
+        url = meta.streamUrls[0].replace("file.mkv", "")
```

Comment:
I think it's worth a comment so that future readers know why we're doing this :smile:.

```suggestion
        # Kodi will try to look for subtitles in the directory containing the file. '/' and `/file.mkv` both point
        # to the file, and Kodi will happily try to read the whole file without recognizing it isn't a directory.
        # To get around that, we omit the filename here since it is unnecessary.
        url = meta.streamUrls[0].replace("file.mkv", "")
```

Reply:
I'd suggest replacing "Kodi" with "Kodi 19" as that's a specific behaviour.